contracts/tests/sbtc-withdrawal.test.ts

```diff
@@ -233,14 +233,40 @@ describe("initiating a withdrawal request", () => {
   });
 });
 
+test("max-fee must be accounted for", () => {
+  txOk(
+    deposit.completeDepositWrapper({
+      txid: new Uint8Array(32).fill(0),
+      voutIndex: 0,
+      amount: 4000n,
+      recipient: alice,
+    }),
+    deployer
+  );
+  const receipt = txErr(
+    withdrawal.initiateWithdrawalRequest({
+      amount: 4000n,
+      recipient: alicePoxAddr,
+      maxFee: 1000000n,
+    }),
+    alice
+  );
+  // Under the hood `initiate-withdrawal-request` attempts `ft-burn?`
+  // amount + max-fee for the `tx-sender`, so if the transaction sender
+  // does not have enough in their account then `(err u1)` is returned in
+  // the response.
+  expect(receipt.value).toEqual(1n);
+});
+
+
 describe("Accepting a withdrawal request", () => {
   test("Fails with non-existant request-id", () => {
     // Alice initiates withdrawalrequest
     txOk(
       deposit.completeDepositWrapper({
         txid: new Uint8Array(32).fill(0),
         voutIndex: 0,
-        amount: 1000n,
+        amount: 1000n + 10n,
```

Comment:
I think it would help readability in general if these were named variables, like `defaultAmount` and `defaultMaxFee` which were used throughout (then a `defaultAmount + (defaultMaxFee + 10n)` would be pretty obvious when you want to deviate from it).  Just a thought.

Reply:
Yeah I should have done this better than I did. I'll update this to something better.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/498/commits/c43a8307c566c2156741864a0c2a730f4dfab7d3.